README.md

```diff
@@ -332,6 +332,21 @@ const const player = { name: "Lu" }!
 print("Hello {player$name}!")!
 ```
 
+## String operations
+
+You can use the `substring` method to make strings shorter:
+
+```java
+'Hello world!'.substring(2, -3) // 'llo wor'
+```
+
+Inversely, you can use the `superstring` method to make strings longer:
+
+```java
+'llo wor'.superstring(-2, 3) // 'Hello world!'
+```
```

Comment:
It's a lot to write, can't you just do `'Hello world!'[2:-3]` and `'llo wor'[-2:3]` instead ?